File: applicatie/src/main/java/com/vdzon/administratie/pdfgenerator/pdfutil/PdfData.java
package com.vdzon.administratie.pdfgenerator.pdfutil;

import org.apache.pdfbox.pdmodel.PDPageContentStream;
import org.apache.pdfbox.pdmodel.font.PDFont;
import org.apache.pdfbox.pdmodel.font.PDType1Font;

public class PdfData {
    public PDFont fontPlain = PDType1Font.HELVETICA;
    public PDFont fontBold = PDType1Font.HELVETICA_BOLD;
    public float pos = 0;
    public float pageHeight = 0;
    public float pageWidth = 0;
    public PDPageContentStream page;

}

File: pdfgenerator/impl/src/main/java/com/vdzon/administratie/pdfgenerator/pdfutil/PdfData.kt
package com.vdzon.administratie.pdfgenerator.pdfutil

import org.apache.pdfbox.pdmodel.PDPageContentStream
import org.apache.pdfbox.pdmodel.font.PDFont
import org.apache.pdfbox.pdmodel.font.PDType1Font

class PdfData {
    var fontPlain: PDFont = PDType1Font.HELVETICA
    var fontBold: PDFont = PDType1Font.HELVETICA_BOLD
    var pos = 0f
    var pageHeight = 0f
    var pageWidth = 0f
    var page: PDPageContentStream? = null

}
